pixi.toml

```diff
@@ -235,6 +235,65 @@ js-build-all = { cmd = "yarn --cwd rerun_js workspaces run build", depends_on =
   "js-install",
 ] }
 
+[feature.wheel-build.tasks]
+# Build and install a development version of the rerun-sdk Python package.
+#
+# This only needs to be called when you have made changes that would impact the rust bindings of
+# the python package. The python code will be imported directly from the source folder and will
+# reflect changes immediately upon re-import.
+# Note:
+# - pip (which is called by maturin develop) doesn't recognize conda/pixi envs as venv, and thus complains if
+#   configured to not install outside venv (which is a good practice). PIP_REQUIRE_VIRTUALENV=0 disables this check.
+# - RERUN_ALLOW_MISSING_BIN is needed to allow maturin to run without the `rerun` binary being part of the rerun-sdk
+#   package.
+py-build-common = { cmd = "PIP_REQUIRE_VIRTUALENV=0 RERUN_ALLOW_MISSING_BIN=1 maturin develop --manifest-path rerun_py/Cargo.toml --extras=tests", depends_on = [
+  "rerun-build", # We need to build rerun-cli since it is bundled in the python package.
+] }
+
+[feature.python-tasks.tasks]
+
+py-build = "pixi run -e py py-build-common"
```

Comment:
This causes problems when running `docs/snippets/compare_snippet_output.py`, since it build the python wheel to a different environment than the one running the snippets.

We either need to add `-e py` to all python invocations in `compare_snippet_output.py` (but I think that will break CI, which calls `compare_snippet_output.py` with `-e wheel-test`), or call `py-build-common` from `compare_snippet_output.py` instead of `py-build`. But then I wonder - what is the `py-build` command for?

Setting the environment inside the environment like this seems like an inherently bad idea since it means the environment the user asked for will be ignored